bin/collator/src/cli.rs

```diff
@@ -103,3 +144,37 @@ impl RelayChainCli {
         }
     }
 }
+
+#[derive(Debug, PartialEq, Clone)]
+pub enum EthApi {
+    Debug,
+    Trace,
+}
+
+impl std::str::FromStr for EthApi {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        Ok(match s {
+            "debug" => Self::Debug,
+            "trace" => Self::Trace,
+            _ => {
+                return Err(format!(
+                    "`{}` is not recognized as a supported Ethereum Api",
+                    s
+                ))
+            }
+        })
+    }
+}
+
+pub struct TracingConfig {
```

Comment:
How about `EvmTracingConfig`? It's more precise.

Reply:
Done